packages/phoenix-evals/src/phoenix/evals/models/anthropic.py

```diff
@@ -93,8 +96,8 @@ def _init_rate_limiter(self) -> None:
         self._rate_limiter = RateLimiter(
             rate_limit_error=self._anthropic.RateLimitError,
             max_rate_limit_retries=10,
-            initial_per_second_request_rate=1,
-            maximum_per_second_request_rate=20,
+            initial_per_second_request_rate=self.initial_rate_limit,
+            maximum_per_second_request_rate=self.initial_rate_limit * 4,
```

Comment:
4 means arbitrary. Why not just parameterize the max too so the function is more pure?

Reply:
It's not truly arbitrary, our ramp up corresponds to approximately a doubling of the rate approximately every minute and is tuned to try and stabilize around a steady-state of errors per second. The cap is there to prevent unreasonable runaway—for the time being I think we might want to not expose this to avoid the complexity of explaining the details of the feedback we use to control our throttler

Reply:
I pushed this logic and justification down into our throttler